quickstarts/contentsquare/config.yml

```diff
@@ -0,0 +1,69 @@
+# Name of the quickstart (required)
+name: contentsquare
+
+# Displayed in the UI (required)
+title: Contentsquare
+
+# Displayed in search results and recommendations. Summarizes a quickstarts functionality.
+summary: |
+  Contentsquare is a powerful, yet easy-to-use, digital experience analytics 
+  platform that collects and quantifies user behavior across all your digital 
+  properties showing you what users like, what they don’t like, where they 
+  struggle, and why they leave. 
+  Contentsquare surfaces opportunities to improve your customer experience
+  automatically and easily so you can increase your conversion rates, improve
+  customer loyalty, reduce costs and drive revenue.
+
+# Long-form description of the quickstart (required)
+description: |
+  Get more value from your investments in Contentsquare and New Relic by seeing
+  how performance issues truly affect your customers’ digital experience. 
+
+  Through this integration, access Contentsquare Session Replay links directly
+  within New Relic. With this, Contentsquare shows you exactly what a user was 
+  doing before, during, and after an issue occurs, so that you can remediate 
+  issues quickly and with the complete picture. Dramatically reduce the 
+  time-to-repair of system errors by linking to your impacted users' exact 
+  session replays.
+
+  - <b>Reduce time-to-recovery</b>
+    By integrating Contentsquare with your New Relic, you can provide your 
+    engineering team with all of the necessary technical and behavioral data 
+    needed to identify the root cause of an issue without having to spend cycles 
+    trying to recreate the issue.
+
+
+  - <b>Prioritize the issues that matter most</b>
+    Combine the technical data from New Relic with the behavioral data surfaced 
+    from Contentsquare to uncover and prioritize the issues that have the
+    greatest impact on your revenue and customer experience metrics.
+
+# Support level: New Relic | Verified | Community (required)
+level: Community
+
+# Authors of the quickstart (required)
+authors:
+  - Contentsquare
+
+# Keywords for filtering / searching criteria in the UI
+keywords:
+  - Behavioural Analytics
+  - Customer Experience
+  - Digital Experience
+  - User Experience
+
+documentation:
```

Comment:
The first documentation URL listed in the documentation configuration should be the primary doc reference. it will the one that will be invoked when the user click on  the `installation docs` button.  I just wanted to check to see if these are in the correct order.

https://github.com/newrelic/newrelic-quickstarts/blob/main/CONTRIBUTING.md#documentation

Reply:
Thank you! Not the right order then, I will update that